<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>qnlMain</class>
 <widget class="QDialog" name="qnlMain" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Qt-Nlist Version 0.47</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QListWidget" name="listFilename" />
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <layout class="QVBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QPushButton" name="refButton" >
             <property name="text" >
              <string>Ref</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="delButton" >
             <property name="text" >
              <string>Delete</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Mode</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QRadioButton" name="radioNetlist" >
            <property name="text" >
             <string>&amp;Netlist</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioPartslist" >
            <property name="text" >
             <string>&amp;Partslist</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Output</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditOutput" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkNotListNetNOCONNECTION" >
     <property name="text" >
      <string>Eliminate signal "NOCONNECTION"</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Report</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditReport" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="aboutButton" >
       <property name="text" >
        <string>About</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="executeButton" >
       <property name="text" >
        <string>Execute</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quitButton" >
       <property name="text" >
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>refButton</tabstop>
  <tabstop>delButton</tabstop>
  <tabstop>radioNetlist</tabstop>
  <tabstop>radioPartslist</tabstop>
  <tabstop>lineEditOutput</tabstop>
  <tabstop>checkNotListNetNOCONNECTION</tabstop>
  <tabstop>lineEditReport</tabstop>
  <tabstop>quitButton</tabstop>
  <tabstop>executeButton</tabstop>
  <tabstop>aboutButton</tabstop>
  <tabstop>listFilename</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>executeButton</sender>
   <signal>clicked()</signal>
   <receiver>qnlMain</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>qnlMain</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
